### PR TITLE
Bluespace And Normality Ores

### PR DESCRIPTION
**Pull request by VMSolidus**
# Description

This PR adds new ores in the form of Bluespace and Normality ores, the former being just as rare as artifact fragments, and the latter being even rarer still. These ores can be mined by salvage, will generate on asteroids or expeditions, and can be processed in an ore processor into bluespace or normality crystals, which Epistemics can use to create psionic related objects. Note that the two are currently indistinguishable from the other until you actually mine them. 

Yes this DOES infact make it easier to make glimmer drains and probers. I need this as an excuse to make psionics even more dangerous. 

<details><summary><h1>Media</h1></summary>
<p>

image

image

image

</p>
</details>

# Changelog

:cl:
- add: Added Bluespace and Normality ore as very rare ore types. Salvage can occasionally find these ores on Asteroids or on Expedition planets. Bluespace and Normality Ore can be smelted in an Ore Processor into Bluespace and Normality Crystals. Make sure to bring these to Epistemics so that they can do their job.

### CODE REVIEW

File: Resources/Locale/en-US/materials/materials.ftl [hunk 1]
```diff
@@ -35,6 +35,8 @@ materials-raw-plasma = raw plasma
 materials-raw-uranium = raw uranium
 materials-raw-bananium = raw bananium
 materials-raw-salt = raw salt
+materials-raw-bluespace = raw bluespace
+materials-raw-normality = raw normality
```
Comment (DEATHB4DEFEAT) [inline Resources/Locale/en-US/materials/materials.ftl:39]: The name simply being "normality" seems really odd to me

File: Resources/Prototypes/Nyanotrasen/Stacks/materials.yml [hunk 2]
```diff
@@ -5,3 +5,25 @@
...
+  spawn: MaterialNormality
+  maxCount: 5
+  itemSize: 1
+
+- type: stack
+  id: NormalityOre
+  name: rough normality
+  icon: { sprite: Objects/Materials/ore.rsi, state: bluespace }
+  spawn: NormalityOre1
+  maxCount: 30
```
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/Nyanotrasen/Stacks/materials.yml:29]:
```suggestion
  maxCount: 30

```

File: Resources/Prototypes/Procedural/salvage_loot.yml [hunk 3]
```diff
@@ -185,3 +185,17 @@
...
+  loots:
+    - !type:BiomeMarkerLoot
+      proto: OreBluespace
+
+- type: salvageLoot
+  id: OreNormality
+  guaranteed: true
+  loots:
+    - !type:BiomeMarkerLoot
+      proto: OreNormality
```
Comment (DEATHB4DEFEAT) [inline Resources/Prototypes/Procedural/salvage_loot.yml:201]:
```suggestion
      proto: OreNormality

```

**Review by DEATHB4DEFEAT — COMMENTED** [review]
stop not leaving a trailing newline

**Comment by VMSolidus** [comment]
> stop not leaving a trailing newline

I HAVE NO IDEA WHY IT'S NOT.

**Review by OldDanceJacket — APPROVED** [review]

**Review by Peptide90 — APPROVED** [review]

**Pull request merged**